doc/src/inputs/EditableLabel.mdx

```diff
@@ -0,0 +1,34 @@
+---
+name: EditableLabel
+route: /EditableLabel
+menu: Inputs
+---
+
+import { Playground, Props } from 'docz';
+import { EditableLabel } from '@cosmotech/ui';
+
+# EditableLabel
+
+The aim of this component is to display an editable label whose value can be checked
+
+## Props
+
+<Props of={EditableLabel} />
+
+## Basic Usage
+
+<Playground>
+  <EditableLabel
+    value="Foo"
+    onProcess={() => console.log('New value accepted')}
+    checkValue={(value) => {
+      if (value.length === 0) {
+        return 'TextField cannot be empty';
+      }
+      return null;
+    }}
+    labels={{
+      label: 'TextField',
+    }}
```

Comment:
Props names have changed, don't forget to update the .mdx file :slightly_smiling_face: